docs/security/faq.md

```diff
@@ -0,0 +1,188 @@
+
+**Important: the following is based on possibly limited information. If you know of a similar project with a security feature
+mentioned here that's not attributed, please report.**
+
+* This is the only project with a clear policy on forks.
+* To my knowledge this project is the only one that uses `btc-rpc-proxy` to protect `bitcoind` out-of-the-box.
+* This project was the first one to implement any kind of RPC filtering for `bitcoind`, long before it was available in `bitcoind` itself.
+  This was later implemented in `nix-bitcoin`, which is the only other project that does this.
+* This project was the first to implement a special macaroon that doesn't allow BTCPayServer to steal sats from LND.
+  This was later copied by `nix-bitcoin` and it's the only other project that does this.
+  Note that this is meant as a compliment to `nix-bitcoin` - copying important security features is a very good thing!
+* This project implements SSO for several applications.
+  The only other known project that does this is BTCPayServer-docker.
+  The application handling authentication is intentionally much, much simpler than BTCPayServer, thus the attack surface is lower.
+* This project sandboxes cookeis of different apps to avoid them being passed between each-other.
+  There's no other known project that does this.
+* The keys of foreign developers are verified using memory-safe GPG implementation [`sqck`](https://github.com/Kixunil/sqck).
+  No other known project does this.
+* All official builds are performed in a dedicated Qubes VM.
+  No other known project does this.
+* Perhaps except `nix-bitcoin`, no other project attempts deterministic builds.
+  (Only ThunderHub is undeterministic here.)
+* The only other known project that uses declarative approach is `nix-bitcoin`.
+
+## Is it good idea to put my lifesavings into this node?
+
+No.
+
+## Why doesn't this project use Docker?
+
+Docker was **not** designed as a security mechanism and shouldn't be used as such.
+There were actually numerous cases of escaping from Docker containers.
+The Docker daemon adds its own attack surface.
+
+That being said, Docker is just a collection of several mechanisms, some of which may be beneficial for security.
+This project already uses some of them and maybe it will use more in the future (needs to be weighted against disadvantages).
+
+## Does this project use virtual machines?
+
+No, while VMs are a great technology when used properly, such as in the case of Qubes OS, they are heavy.
+It could have negative effect of discouraging users from using it and leaving them with much worse options.
+Finding a sweet spot can be more valuable.
+
+This doesn't mean that VMs will **not** be researched in the future as an optional mechanism.
+It's definitely considered a wishlist item.
+
+## So how does this project isolate services?
+
+Using different (system) users.
+One for each service, except where it makes zero sense.
```

Comment:
a bit more detail would be nice. not just that it generates new users, but also how it applies the least privilege.

Reply:
otoh, it's explained in more detail in the next question...

Reply:
Expanded a little bit.